### PR TITLE
2467: Delete offers tab

### PR DESCRIPTION
**Pull request by steffenkleinle**
### Short description

Delete the offers tab and purge all related components, files, types, screenshots and icons.

### Proposed changes

- Remove the dedicated route for offers and the corresponding types
- Remove all components related
- Remove the `postData` from `OfferModel` as we only support internal offers now (and set the zammad-url for the malteHelpOffer as path instead)
- Remove screenshots with offers

### Side effects

- Old offer URLS do not work anymore

### Resolved issues

Fixes: #2467.

---

### CODE REVIEW

**Review by LeandraH — COMMENTED** [review]
Wow, thanks for slogging through that! Looks good already but in native I get an error about the imports of two `Offers.svg` files in `native/src/constants/buildConfig.ts`. Once I delete those, it looks good. (Also tested on Firefox)

I also think that the docs for `i18n` and `whitelabelling`should not mention the offers anymore.

**Comment by steffenkleinle** [comment]
> Wow, thanks for slogging through that! Looks good already but in native I get an error about the imports of two `Offers.svg` files in `native/src/constants/buildConfig.ts`. Once I delete those, it looks good. (Also tested on Firefox)

Forgot to commit those o.O Good catch!

> I also think that the docs for `i18n` and `whitelabelling`should not mention the offers anymore.

done

File: web/.idea/prettier.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="PrettierConfiguration">
+    <option name="myConfigurationMode" value="AUTOMATIC" />
```
[the remark this answers is not in the document]
Comment (steffenkleinle) [reply web/.idea/prettier.xml:4]: I think automatic prettier configuration won't hurt, right? I'd keep it if there are no objections.

**Review by ztefanie — APPROVED** [review]
Looks good, tested Android works as expected.

**Pull request merged**